gk/main.c

```diff
@@ -542,6 +607,87 @@ setup_gk_instance(unsigned int lcore_id, struct gk_config *gk_conf)
 	return ret;
 }
 
+static struct flow_entry *
+find_flow_entry_candidate(struct gk_instance *instance,
+	uint32_t bidx, uint64_t request_timeout_cycles,
+	enum gk_flow_state state)
+{
+	int32_t index;
+	uint32_t next = 0;
+	const struct ip_flow *key;
+	struct flow_entry *last_fe = NULL;
+	void *data;
+	time_t now = time(NULL);
+	RTE_VERIFY(now >= 0);
```

Comment:
1. Change the definition of `now`  to `uint64_t now = rte_rdtsc();`.

2. Drop the `RTE_VERIFY()`.